README.md

```diff
@@ -1,3 +1,30 @@
+# Horned Beasts
+
+
+
+
```

Comment:
- We just need to add in a contributors section into the readme and then it is good to go (see lab 1 feedback) 👍